### DISCUSSION TITLE
Error when trying to build project

### DISCUSSION THREAD

**Discussion by Czellmer1324** [Petersoj/alpaca-java#163] [Q&A]
![image](https://github.com/user-attachments/assets/c394e2dc-c4de-420f-9ffe-812266b3f4a6)
I have having this error pop up in intelij when trying to build this project. In addition I am having errors pop up saying it cannot recognize the symbols in the openapitools import statement at the top of the build.gradle file. Any help would be appreciated!

**Comment by Petersoj** [comment]
Thanks for reporting this issue. This is a simple fix that I will address soon.
